src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Contact {
+	
+	public final String value;
+	private boolean isPrivate;
+
+	public Contact(String contact, boolean isPrivate, String CONTACT_MESSAGE_CONSTRAINTS, String CONTACT_VALIDATION_REGEX) throws IllegalValueException {
```

Comment:
Avoid lines longer than 110 chars. When a line is too long, it requires horizontal scrolling, that slows down the reader/reviewer.